go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/logrusorgru/aurora v2.0.3+incompatible
 	github.com/onflow/cadence v1.0.0-preview.38
 	github.com/onflow/crypto v0.25.1
-	github.com/onflow/flow-core-contracts/lib/go/templates v1.3.0
+	github.com/onflow/flow-core-contracts/lib/go/templates v1.3.1
```

Comment:
Why are the core contracts updated? This should match what is in flow-go. If the update is needed, flow-go should be updated first

Reply:
it's updated here:
https://github.com/onflow/flow-go/pull/6004/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R53-R54